src/components/loading-indicator/__tests__/makeLoadable.test.tsx

```diff
@@ -35,7 +40,8 @@ describe('components/loading-indicator/makeLoadable', () => {
 
     test('should pass loadingIndicatorPorps to LoadingIndicator', () => {
         const LoadableComponent = makeLoadable(TestComponent);
-        const wrapper = shallow(<LoadableComponent isLoading loadingIndicatorProps={{ className: 'foobar' }} />);
+        // eslint-disable-next-line prettier/prettier
```

Comment:
Not yet sure how to fix this lint error

Reply:
May need to change the parser option to `typescript` https://prettier.io/docs/en/options.html#parser

Reply:
Good call, that worked